README.md

```diff
@@ -226,6 +226,15 @@ wish to tweak it, take a look at the documentation for [Rihanna.Config](https://
 
 Please refer to the [Changelog](CHANGELOG.md) for details on when a database upgrade is required and how to migrate the Rihanna jobs table.
 
+If you're on Rihanna v2, please check your `rihanna_jobs_locking_index` and check `due_at`. If it does not have `NULLS FIRST`, you'll want to
```

Comment:
Nobody is gonna read this. Any way we can make it automatic?

Reply:
1. Every upgrade requires someone to read something, even if it's just that they need to create a migration
2. We could try to introspect the index on the table to see if it has NULLS FIRST, but I'm not sure that's possible
3. Would we want to raise an error on startup after this upgrade? I know that in the past rihanna wouldn't work when columns were missing.
4. We definitely don't want to make fully automatic changes, even with CONCURRENT, but I'm not sure that's what you're suggesting.

Reply:
I'm open to suggestions for how you'd like to improve this. Maybe there is a better doc for it to go into.

Reply:
Ideally we would detect this

Reply:
If I can detect it, what do you want to do with that information? Warning, error, something else?

Reply:
This might work:

```
SELECT indexdef
FROM pg_indexes
WHERE tablename = '{tablename}'
AND indexname = '{tablename}_locking_index'
```

Then we could check for `NULLS FIRST` in the `indexdef`.

Reply:
I like this approach

Reply:
Cool. I have a baby coming in two days, so I'm not sure when I'll be able to finish this. I'm happy to let someone else take over from here.